src/main/java/uk/ac/ebi/eva/utils/FileUtils.java

```diff
@@ -74,4 +78,21 @@ public static Properties getPropertiesFile(InputStream propertiesInputStream) th
         properties.load(propertiesInputStream);
         return properties;
     }
+
+    /**
+     * Creates a temporary GzipFile withe the content at {@param content}.
+     * @param content
+     * @param name how the temporal file will be called
+     * @return
+     * @throws IOException
+     */
+    public static File newGzipFile(String content, String name) throws IOException {
```

Comment:
Yes discussed already with @jorizci and decided to duplicate the logic for now...